src/Components/AddComment.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+import postComment from "../../utils/add-comment";
+import getCommentsById from "../../utils/get-comments-by-id";
+
+function AddComment(props) {
+  const { articleId } = props;
+  const [username, setUsername] = useState("");
+  const [comment, setComment] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComment(articleId, comment, username).then((response) => {
+      if (response) {
+        setUsername("");
+        setComment("");
+        getCommentsById();
+        alert("Comment posted successfully");
+      }
+    });
```

Comment:
When we post a comment, it should appear in the list of comments without requiring a refresh - this will mean you'll need to update the state of the list of comments 🔍